.circleci/config.yml

```diff
@@ -90,11 +90,11 @@ jobs:
           name: Build
           command: cd build && make -j2
       - run:
-          name: Build tests
-          command: cd build && make -j2 test
+          name: Reset databases
+          command: cd build && make -j2 db
```

Comment:
`-j2` means "use two threads" -- you don't need two threads to do this; it won't help